### DISCUSSION TITLE
White and Black alpha colors

### DISCUSSION THREAD

**Discussion by RainBoard01** [mantinedev/mantine#1546] [Feature requests]
Could be great and handful, a section in the default color theme for the transparencies, just like the Alphas of ChakraUI.

Thank you for this awesome library! 😊

**Comment by rtivital** [comment]
you can set `theme.white` and `theme.black` to whatever colors you want

**Reply by fer-ri** [reply]
Can we reference value for `theme.black` from `theme.colors`? lets say I want to set into `theme.colors.gray[8]`

Thanks
